### DISCUSSION TITLE
MiniO Upload: "the region is wrong; expecting home"

### DISCUSSION THREAD

**Discussion by dom6770** [outline/outline#5992] [Self Hosting]
I got this weird behavior which I couldn't find anywhere here. 

I can't tell for certain how long this issue persisted. Outline is only used by me as a personal wiki, and after months I wanted to change the avatar. Every time I tried, I got "Upload failed" in the bottom left corner, and in the Chrome developer console, this:

`POST https://api.minio.example.com/outline 400 (Bad Request)`

When I switch to the "Network" tab, I can further investigate the XmlHttpRequest, which display me this as a response:

```
<?xml version="1.0" encoding="UTF-8"?>
<Error>
  <Code>AuthorizationHeaderMalformed</Code>
  <Message>The authorization header is malformed; the region is wrong; expecting &#39;home&#39;.</Message>
  <BucketName>outline</BucketName>
  <Resource>/outline</Resource>
  <Region>home</Region>
  <RequestId>178D21E11AEF25B2</RequestId>
  <HostId>dd9025bab4ad464b049177c95eb6ebf374d3b3fd1af9251148b658df7ac2e3e8</HostId>
</Error>
```

This not only affected uploading avatars, but also images to documents.

Odd, why does Outline expect that the `AWS_REGION` is set to `home`? It was set to `local`, after switching it to `home` the upload is successful. This is nowhere documented. Am I the only one with this issue? 

Outline Version: 0.72.1
MiniO Version: 2023-10-07T15:07:38Z

**Comment by tommoor** [comment]
That error is from Minio, not Outline – there's no expectations around the bucket name.
